src/oca_github_bot/tasks/merge_bot.py

```diff
@@ -141,7 +141,10 @@ def _merge_bot_merge_pr(org, repo, merge_bot_branch, cwd, dry_run=False):
         d for d in modified_addon_dirs if is_addon_dir(d, installable_only=True)
     ]
 
-    # update HISTORY.rst using towncrier, before generating README.rst
+    # Update HISTORY.rst using towncrier, before generating README.rst.
+    # We don't do this if nobump is specified, because updating the changelog
+    # is something we only do when "releasing", and patch|minor|major is
+    # the way to mean "release" in OCA.
```

Comment:
does this means that - say - 13.0.1.0.0 is not added to the history?

Reply:
@simahawk Yes. There is https://github.com/OCA/oca-github-bot/issues/54 to discuss the special case of migration PRs.